controllers/suggestions/suggest_event_webcast_controller.py

```diff
@@ -13,7 +13,7 @@ class SuggestEventWebcastController(LoggedInHandler):
     """
 
     def get(self):
-        self._require_login("/suggest/event/webcast?event=%s" % self.request.get("event_key"))
```

Comment:
It does. `_require_login()` will always redirect to `self.request.url` unless otherwise specified. I see no use case for it being specified since everywhere it was being used was redirecting back to itself but I kept it in just in case.